builds/azure-pipelines/build-release.yml

```diff
@@ -88,7 +90,7 @@ stages:
     displayName: Build Java Library
 
     pool:
-      vmImage: 'ubuntu-latest'
+      vmImage: 'windows-latest'
```

Comment:
Changed vmImage to windows because CodeQL with Guardian is not yet supported on Linux.